tests/unit/type/list.tcl

```diff
@@ -553,6 +553,8 @@ foreach {type large} [array get largevalue] {
     foreach resp {3 2} {
         if {[lsearch $::denytags "resp3"] >= 0} {
             if {$resp == 3} {continue}
+        } elseif {$::force_resp3} {
+            if {$resp == 2} {continue}
```

Comment:
maybe with the recent improvements you don't need these and can let the RESP2 tests run as well?
same for zset.tcl

Reply:
some tests use rawread to verify the line protocol, in these cases we do not intervene and just skip what's needed if force_resp3 (only the ones that test resp2)